test/get.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 var throws = require('assert').throws;
-var vm = require('vm');
 
 var eq = require('./utils').eq;
 var errorEq = require('./utils').errorEq;
 var S = require('..');
+var T = function() { return true; };
+var F = function() { return false; };
```

Comment:
These functions should be unary rather than nullary, to match the expected type. Perhaps `S.K(true)` is short enough?